src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Confirm.tsx

```diff
@@ -598,7 +574,7 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
             </Box>
           </Row.Value>
         </Row>
-        {!isTokenWithdraw && !isRunePool && (
+        {!isRunePool && (
```

Comment:
Nice catch too, iirc historically we didn't need dust for native asset withdraws but did need some for tokens, then we effectively started requiring it for all as part of upstream requirements, which we never reflected in the UI